### ISSUE TITLE
Improve JDBC configuration documentation

### ISSUE THREAD

**Issue by vpavic** [spring-projects/spring-session#1011]
We should improve our reference documentation on JDBC configuration capabilities to cover more advanced configuration scenarios (including `@SpringSessionDataSource` and other means of configuration customization).

**Comment by gotson** [comment]
As commented [here](https://github.com/spring-projects/spring-session/issues/1469#issuecomment-918927410), it would be good to add a section in the document on how to disable transactions for Spring Session JDBC.

> This is now resolved by adding configuration support for picking up TransactionOperations bean named springSessionTransactionOperations. Coupled with the new transaction-less TransactionOperations factory that will be available in Spring Framework 5.2, disabling transactions for JdbcOperationsSessionRepository is as simple as:

```java
@Bean
public TransactionOperations springSessionTransactionOperations() {
    return TransactionOperations.withoutTransaction();
}
```

SQLite performance using transactions is very bad, because it only supports `SERIALIZABLE` level of transactions. This configuration should help people with SQLite and JDBC sessions.

**Comment by marcusdacoregio** [comment]
Closed via https://github.com/spring-projects/spring-session/issues/2827
